.gitignore

```diff
@@ -32,6 +32,8 @@ obj/**
 .pytest_cache
 temp/**
 tmp/**
+src/test/datascience/tmp/**
+src/test/datascience/temp/**
```

Comment:
Moved notebooks created into a directory within the test workspace folder.
Python extension doesn't seem to use workspace active interpreter as active interpreter for files outside the workspace. Seems to be a recent change (our test open the workspace folder `src/test/datascience`).